docker/jenkins/build.sh

```diff
@@ -32,4 +32,7 @@ make -j${NPROCS}
 
 ctest --no-compress-output -T Test
 
+cd ${ARBORX_DIR}
+CLANG_FORMAT_EXE="clang-format" ./scripts/check_format_cpp.sh
```

Comment:
Actually what about running the format check 1st?

Reply:
That would mean that you won't see the results of the tests if your formatting fails. Formatting is secondary.